guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,6 +16,7 @@ Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
 For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+* Ensure that you have at least the same amount of free space as that consumed by `/var/lib/pgsql/data` for PostgreSQL upgrade.
```

Comment:
This should be RPM-only and use the attribute we already have
```suggestion
ifndef::foreman-deb[]
* Ensure that you have at least the same amount of free space as that consumed by `{postgresql-data-dir}` for PostgreSQL upgrade.
endif::[]
```

Reply:
@ekohl hi, Instead of saying "some amount of free space" can we provide any rough estimate about the required free space?  Like if the existing size of /var/lib/pgsql/data is 15G then you need at least XX G free on the partition. As some customers have dedicated partition for /var/lib/pgsql/ .

Reply:
@ahumbe it's the same amount because it's literally copied to a backup location, so in your example it'd be at least 15G free.

We have implemented this in a technical check as well: our installer hook checks exactly that in https://github.com/theforeman/foreman-installer/blob/67702be227056c29bbe9e34089009d69bb3a6758/hooks/boot/06-postgresql-upgrade-extensions.rb#L67-L81 and foreman_maintain has it in https://github.com/theforeman/foreman_maintain/blob/master/definitions/checks/disk/available_space_postgresql13.rb.

Now that I read that, I see it uses `/var/lib/pgsql` for the checks so it should be:
```suggestion
ifndef::foreman-deb[]
* Ensure that you have at least the same amount of free space as that consumed by `{postgresql-lib-dir}` for PostgreSQL upgrade.
endif::[]
```